src/stores/activitiesStores/Finalize.js

```diff
@@ -0,0 +1,77 @@
+import { observable, action, runInAction, reaction } from 'mobx';
+import _ from 'lodash';
+import { Token, OracleStatus, AppLocation, SortBy } from '../../constants';
+import { queryAllOracles } from '../../network/graphQuery';
+import Oracle from '../models/Oracle';
+
+const INIT = {
+  loaded: false, // loading state?
+  loadingMore: false, // for laoding icon?
+  list: [], // data list
+  hasMore: true, // has more data to fetch?
+  offset: 0, // skip
+  limit: 16, // loading batch amount
```

Comment:
kinda feel like all these values might be a little overkill. but ok to keep them.